exporter/attestation/unbundle.go

```diff
@@ -28,6 +33,12 @@ func Unbundle(ctx context.Context, s session.Group, bundled []exporter.Attestati
 		eg.Go(func() error {
 			switch att.Kind {
 			case gatewaypb.AttestationKindInToto:
+				if strings.HasPrefix(att.InToto.PredicateType, "https://slsa.dev/provenance/") {
```

Comment:
No need to check `..` in the URL path?

I assume not needed, but posting this for confirmation

Reply:
I don't think we do - my understanding is that end-users should treat as essentially opaque strings and not attempt to resolve them: https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI.

The only reason we do the `strings.HasPrefix` here instead of comparing directly against the strings in the in-toto library is to match against different versions (including unreleased ones).